modules/util/source-formatter/src/test/java/com/liferay/source/formatter/processor/UpgradeSourceProcessorTest.java

```diff
@@ -68,6 +68,11 @@ public void testUpgradeJavaExtractTextMethodCheck() throws Exception {
 		test("upgrade/UpgradeJavaExtractTextMethodCheck.testjava");
 	}
 
+	@Test
+	public void testUpgradeJavaMultiVMPoolUtilCheck() throws Exception {
+		test("upgrade/UpgradeJavaMultiVMPoolUtilCheck.testjava");
```

Comment:
This test case is too simple. We should check that warning message is displayed and that it is displayed for the correct line number. It should look like this:

```java
test(
    "upgrade/XMLUpgradeDTDVersionCheck.testxml",
    "Replace 'TO_BE_REPLACED' with the correct type", 46);   // This is the expected message and the expected line number
```

However, there is an issue with this that will require making changes to `BaseSourceProcessorTestCase.java`. When the test case has an expected message, it will not test for the actual vs. expected contents. We will need to change this behavior so that actual/expected content is checked when necessary.

Reply:
@nicolas-sss, let me know if you have any questions about this and/or if you need help making changes to `BaseSourceProcessorTestCase.java`.

Reply:
I get it, Kevin. If adding the check for message causes the content to be ignored, it is really something to change when you have an expected file. I will take a look at it, and if I have any questions, I'll get back to you. :)